backend/back/src/services/entities/area.entity.ts

```diff
@@ -28,6 +28,10 @@ export default class Area {
 	@Column({ type: "jsonb", default: [] })
 	parametersFormFlow!: ParametersFormFlowFieldDto[];
 
+	@ApiProperty()
+	@Column({ array: true, type: "text", default: [] })
+	parametersReturnFlow!: string[];
```

Comment:
What this is going to do ?

Reply:
This array in db is for the front, who will show the magic params available for the area

Reply:
And what are the magic params exactly ? It's not very self-explanatory